recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.18.0.bb

```diff
@@ -23,13 +20,15 @@ RPROVIDES_${PN}-soup += "${PN}-souphttpsrc"
 PACKAGECONFIG ??= " \
     ${GSTREAMER_ORC} \
     ${@bb.utils.filter('DISTRO_FEATURES', 'pulseaudio x11', d)} \
+    ${@bb.utils.contains('TUNE_FEATURES', 'm64', 'asm', '', d)} \
```

Comment:
Please, if this is required for the new version, please add to the commit. Otherwise, if it is an improvement, it ought to be done as another commit.

Reply:
It is a new meson option on this version.
I will mention this on the commit message